larray/core/array.py

```diff
@@ -3008,6 +3023,7 @@ def with_total(self, *args, **kwargs):
                 lgkey = axis
                 axis = lgkey[0].axis
                 value = res._aggregate(npop[op], (lgkey,))
+                value.set_axes(value.axes[axis], Axis([label], axis.name), inplace=True)
```

Comment:
use value.set_labels instead?
besides, this will break when you use several groups on the same axis, but Travis already told you that :)